### DISCUSSION TITLE
Filter videos

### DISCUSSION THREAD

**Discussion by okcknet** [dailydotdev/daily#1170] [Feature Requests]
I want to be able to filter and hide videos.

**Comment by nimrodkra** [comment]
You can block video content by either clicking on the “More options”
button (tree dots icon) on a video post and then click on “Don’t show video
content”. Alternatively you can go to Feed settings → Content types →
uncheck video.

**Reply by okcknet** [reply]
thank you for detailed information, it helped a lot.
